src/scss/components/_footer.scss

```diff
@@ -0,0 +1,191 @@
+// section-main-padding
+$footer-section-main-padding-top: $padding-top--24;
+$footer-section-main-padding-bottom: $padding-top--24;
+
+// copyright-padding 
+$logo-and-copyright-padding-top: 64px;
+$logo-and-copyright-padding-bottom: 48px;
+
+$color-border-gray:#dcdacb;
+$color-copyright: #73726c;
+
+$margin-right--16: 16px;
+$margin-right--24: 24px;
+
+footer {
+    .footer-banner {
+        .footer-section-banner {
+            padding: $padding--16--32;
+            display: flex;
+            justify-content: space-between;
+            align-items: center;
+            background: $color-white;
+            border-top: $border-size-default solid $color-border-gray;
+
+            .banner-button-container {
+                flex-shrink: 0;
+            }
+        }
+    }
+
+    .footer-operate {
+        background: $color-white;
+        border-top: $border-size-default solid $color-border-gray;
+
+        .ufb-notice--notice-row--3tUOv {
+            .txt {
+                padding: $padding--24--0;
+            }
+        }
+
+        .image-operate {
+            .img-operate {
+                margin-right: $margin-right--24;
+                margin-bottom: $margin-bottom--24;
+                vertical-align: middle;
+
+                &:last-child {
+                    margin-right: 0;
+                }
+            }
+        }
+    }
+
+    .footer-section-main {
+        background: $color-white;
+        border-top: $border-size-default solid $color-border-gray;
+        padding: $padding--0--32;
+        padding-top: $footer-section-main-padding-top;
+
+        .links-and-locale {
+
+            .locale-select-container {
+                padding: 0;
+                order: 1;
+                flex-shrink: 0;
+                margin-left: auto;
+            }
+
+            .unstyled-list {
+                list-style: none;
+                margin: 0;
+                padding: 0;
+                max-width: none;
+            }
+
+            .link-column {
+                flex-basis: 256px;
+                margin-right: $margin-right--16;
```

Comment:
Please define mr-16, it's the same with padding that you had defined it in padding.scss